ground/gcs/src/plugins/config/outputchannelform.cpp

```diff
@@ -86,14 +86,6 @@ OutputChannelForm::OutputChannelForm(const int index, QWidget *parent, const boo
     ui.actuatorLink->setChecked(false);
     connect(ui.actuatorLink, SIGNAL(toggled(bool)), this, SLOT(linkToggled(bool)));
 
-    // Trigger when autopilot is connected
-    connect(this, SIGNAL(autoPilotConnected()), this, SLOT(onAutopilotConnect()));
```

Comment:
No.  The parent already gets an autopilot connect event and has it connected.  We were using that to calculate minimum/maximum values, but now the parent does it when it processes the bank.  We don't need to know down here.